.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+Thank you for working on TiKV! Please read TiKV's [CONTRIBUTING](https://github.com/pingcap/tikv/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+
+## What have you changed? (mandatory)
+
+Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
+- Summarize your change (mandatory)
+- How does this PR work? Need a brief introduction for the changed logic (optional)
+- Describe clearly one logical change and avoid lazy messages (optional)
+- Describe any limitations of the current code (optional)
+
+Please **NOTE** that:
+- Do not assume the code is self-evident
```

Comment:
I think this first "Do not" is not necessary since we are already asking to describe changes.